docs/how-to/create-a-router/configure-traffic-rules.md

```diff
@@ -8,15 +8,31 @@ This step is **optional** and the default behaviour will not discriminate betwee
 
 It is also possible to configure your router such that each request is only dispatched to a subset of the configured routes, based on some request specific conditions. For example, you might have some models, trained on geography-specific data. So, in this scenario, you want to call `mock` if the request contains header `"X-Region: region-A"` or `model-b` if `X-Region` header equals `"region-B"`. It's possible to achieve this by configuring traffic rules on your router.
 
-Traffic rules define which routes should be "activated" for a particular request to your router. Each rule is defined by one or more request conditions and one or more routes that would be activated if the request satisfies the conditions of the rule. Routes that are not part of any traffic rules will be called for each request.
```

Comment:
@terryyylim, could I request you to sneak in this small note (recently added to the internal doc) into this doc here?

<img width="1230" alt="Screenshot 2022-09-27 at 8 07 43 AM" src="https://user-images.githubusercontent.com/23465343/192402164-4a6fc7f2-e16f-4c76-b0da-9b1251c8459c.png">